src/handlers/query-wallet.ts

```diff
@@ -22,6 +22,10 @@ export async function handleCommand(context: Context) {
   await registerWallet(context, walletAddress);
 }
 
+export function unregisterWallet(context: Context) {
+  console.log("unregisterWallet");
```

Comment:
You can probably set it to null.

Reply:
Yes I will just break the relationship from the user to the wallet (sorry still in draft not done yet)